apps/src/templates/codeDocs/MethodSummaryTable.jsx

```diff
@@ -1,33 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import EnhancedSafeMarkdown from '@cdo/apps/templates/EnhancedSafeMarkdown';
-import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import color from '@cdo/apps/util/color';
 import {tableLayoutStyles} from '@cdo/apps/templates/tables/tableConstants';
-import {TextLink} from '@dsco_/link';
 
-export default function MethodSummaryTable({
-  methods,
-  programmingClassLink,
-  openLinkInNewTab = false
-}) {
+export default function MethodSummaryTable({methods, includeLink}) {
   return (
     <table style={{...tableLayoutStyles.table, width: '100%'}}>
       <tbody>
         {methods.map(method => (
           <tr key={method.key}>
             <td style={styles.method}>
               <h3>
-                {method.name}
-                {programmingClassLink && (
-                  <>
-                    {' '}
-                    <TextLink
-                      href={`${programmingClassLink}#method-${method.key}`}
-                      openInNewTab={openLinkInNewTab}
-                      icon={<FontAwesome icon="external-link" />}
-                    />
-                  </>
+                {includeLink ? (
+                  <a
```

Comment:
Could we still use `Link` or `TextLink` here?

Reply:
those only have an `href` attribute and we want an `onclick` so it scrolls within the page instead of adding a url parameter

Reply:
It's up to you but it looks like `Link` does have an `onclick` handler: https://github.com/code-dot-org/dsco_/blob/dfbf490794215c4fd90419ef82c04202fa67605e/packages/link/lib/Link.jsx#L15. It does require href so it might not be the right fit still

Reply:
Since it requires an href I think what I have works better